data/templates/default.docbook5

```diff
@@ -6,8 +6,12 @@ $if(mathml)$
   xmlns:mml="http://www.w3.org/1998/Math/MathML"
 $endif$
   xmlns:xlink="http://www.w3.org/1999/xlink" >
+$if(lang)$ 
+      xml:lang="$lang$"
+$endif$ >
```

Comment:
This doesn't look quite right - this attribute will be placed outside the `>` in line 8, and we'll get an additional `>`.
Also, these attributes should line up.